README.md

```diff
@@ -92,7 +92,7 @@ In the edge device machine:
   ```
   chmod +x /etc/greenboot/red.d/bootfail.sh
   ```
-- for updating run
+- for updating first be sure that nftables.service was started and run
```

Comment:
don't think we need this comment. we should expect that after the upgrade all of the service will be running.
if that's not the case, we should fix the creation of the edge device / kickstart file / blueprint or other method to enable the required service to be running.

Reply:
You're right - opened a relevant task https://issues.redhat.com/browse/ECOPROJECT-370